### PR TITLE
Fixing IR dump code snippet in llvm.md

### PR DESCRIPTION
**Pull request by nlw0**
Fixing llvm IR code dump example.

### CODE REVIEW

File: doc/src/devdocs/llvm.md [hunk 1]
```diff
@@ -94,10 +94,10 @@ process (e.g. because it creates unserializable state). However, the resulting
 It is also possible to dump an LLVM IR module for just one Julia function,
 using:
 ```julia
-f, T = +, Tuple{Int,Int} # Substitute your function of interest here
+fun, T = +, Tuple{Int,Int} # Substitute your function of interest here
 optimize = false
-open("plus.ll", "w") do f
-    println(f, Base._dump_function(f, T, false, false, false, true, :att, optimize))
+open("plus.ll", "w") do file
+    println(file, Base._dump_function(fun, T, false, false, false, true, :att, optimize, :none, params))
```
Comment (Keno) [inline doc/src/devdocs/llvm.md]: `params` isn't defined here. This example should be copy-pastable ideally (can be made so by adding default argument values to `_dump_function` if necessary).

Comment (nlw0) [reply doc/src/devdocs/llvm.md]: That was a copy-paste accident, sorry. Should be good now, and ironically copy-pastable

Comment (nlw0) [reply doc/src/devdocs/llvm.md]: Ok, `params` already has a default, this was just there from the code I took from somewhere else for some reason

**Comment by nlw0** [comment]
I couldn't really test this using e.g. Revise.jl, having to include the file instead. I would appreciate some information on how to properly work on stdlib files.

**Comment by nlw0** [comment]
The current patch should work fine, introducing default values are a better fix, and the example code should now be copy-pastable. Anything else I can do here?

**Pull request merged**